requirements.txt

```diff
@@ -0,0 +1,5 @@
+flask
```

Comment:
This is managed in pyproject.toml using poetry instead of setuptools.

Reply:
Done